.github/workflows/workflow-validate-repo-markdown.yml

```diff
@@ -1,22 +1,18 @@
-name: Validate Repo Markdown
+name: Metaschema Validate Repo Markdown
 on:
   workflow_call:
     inputs:
       ignorePattern:
         description: 'a pattern provided to grep for files/directories to ignore'
         required: false
-        default: '^docs/'
+        default: '^website/'
```

Comment:
@aj-stein-nist  Is this correct?

Reply:
OK thanks Dave for pointing the bitter truth: I can't read!